### PR TITLE
Adding unit test for jira-labels

### PR DESCRIPTION
**Pull request by bsieber-mozilla**
To resolve issue #55

### CODE REVIEW

File: tests/unit/jbi/test_bugzilla.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+# pylint: disable=cannot-enumerate-pytest-fixtures
+import pytest
+
+from src.jbi import bugzilla
+
+
+def test_get_jira_labels_without_whiteboard():
+    test_bug = bugzilla.BugzillaBug(id=0, whiteboard="")
+    jira_labels = test_bug.get_jira_labels()
+    assert jira_labels
```
Comment (leplatrem) [inline tests/unit/jbi/test_bugzilla.py:13]: I would say that this assertion is useless. py.test provides a pretty good output if you compare a falsy value with a list

(same with other tests)

**Review by leplatrem — APPROVED** [review]

File: tests/unit/jbi/test_bugzilla.py (hunk 1 above)
Comment (bsieber-mozilla) [inline tests/unit/jbi/test_bugzilla.py:13]:
```suggestion
```

File: tests/unit/jbi/test_bugzilla.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    assert ["bugzilla"] == jira_labels
+
+
+def test_get_jira_labels_with_space():
+    test_bug = bugzilla.BugzillaBug(id=0, whiteboard="[test whiteboard]")
+    jira_labels = test_bug.get_jira_labels()
+    assert jira_labels
```
Comment (bsieber-mozilla) [inline tests/unit/jbi/test_bugzilla.py:20]:
```suggestion
```

File: tests/unit/jbi/test_bugzilla.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    expected = ["bugzilla", "test.whiteboard", "[test.whiteboard]"]
+    assert expected == jira_labels
+
+
+def test_get_jira_labels_without_space():
+    test_bug = bugzilla.BugzillaBug(id=0, whiteboard="[test-whiteboard]")
+    jira_labels = test_bug.get_jira_labels()
+    assert jira_labels
```
Comment (bsieber-mozilla) [inline tests/unit/jbi/test_bugzilla.py:28]:
```suggestion
```

File: tests/unit/jbi/test_bugzilla.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    expected = ["bugzilla", "test-whiteboard", "[test-whiteboard]"]
+    assert expected == jira_labels
+
+
+def test_get_jira_labels_multiple():
+    test_bug = bugzilla.BugzillaBug(
+        id=0, whiteboard="[test whiteboard][test-no-space][test-both space-and-not"
+    )
+    jira_labels = test_bug.get_jira_labels()
+    assert jira_labels
```
Comment (bsieber-mozilla) [inline tests/unit/jbi/test_bugzilla.py:38]:
```suggestion
```

File: tests/unit/jbi/test_bugzilla.py (hunk 1 above)
Comment (bsieber-mozilla) [reply tests/unit/jbi/test_bugzilla.py]: https://github.com/mozilla/jira-bugzilla-integration/pull/59#discussion_r871825582

resolving

**Pull request merged**